백엔드 의존성 누락

## 어떤 버그인가요?

- `Error: Cannot find module 'prosemirror-view'` 발생

## 어떤 상황에서 발생한 버그인가요?

- `develop`에서 `yarn docker:dev` 실행 시 `Error: Cannot find module 'prosemirror-view'` 발생

## 예상 결과

- 정상적으로 모든 컨테이너 실행

--- a/apps/backend/package.json
+++ b/apps/backend/package.json
@@ -43,6 +43,7 @@
     "passport-naver": "^1.0.6",
     "path": "^0.12.7",
     "pg": "^8.13.1",
+    "prosemirror-view": "^1.37.0",
     "reflect-metadata": "^0.1.13",
     "rxjs": "^7.8.1",
     "socket.io": "^4.8.1",

